### ISSUE TITLE
Broke a bottom left button, what do I do?

### ISSUE THREAD

**Issue by ekkofox** [sqfmi/Watchy#39]
When assembling my watchy and putting it into the case, I ended up somehow breaking the bottom left button. I have the little plastic piece but it will not stay in. Either I need to do micro dots of glue or try to get a new pcb. Is there a way to get just the pcb part? I don't want to have to get a whole other kit. I just want to know what can I do in this case?

**Comment by sqfmi** [comment]
Hi @ekkofox , do you have a picture of how it is broken? The easiest way is to replace the button if you are handy with soldering. If not, we can order a new PCB for you. Reach out to us vai e-mail or DM us on Discord and we can look into it, thanks!

**Comment by raywin27** [comment]
I just got my watchy, and have a broken SW2, it was DOA,  i'd like to try and replace the button, I found them here: https://lcsc.com/product-detail/Tactile-Switches_Korean-Hroparts-Elec-K2-1114SA-A4SW-06_C136662.html, but I think it is kind of crazy to order just one.  I'll try and email or dm you on discord

**Comment by ekkofox** [comment]
![photo_2021-06-11_15-54-48](https://user-images.githubusercontent.com/85705940/121742175-7c73c380-cacd-11eb-9554-83358a0218d0.jpg)
![photo_2021-06-11_15-54-45](https://user-images.githubusercontent.com/85705940/121742184-80074a80-cacd-11eb-90f8-25136238f0ed.jpg)
![photo_2021-06-11_15-54-42](https://user-images.githubusercontent.com/85705940/121742192-81d10e00-cacd-11eb-9f03-1fd587cb0449.jpg)
![photo_2021-06-11_15-55-29](https://user-images.githubusercontent.com/85705940/121742203-85fd2b80-cacd-11eb-9af1-6de660bc59bf.jpg)

**Comment by ekkofox** [comment]
> Hi @ekkofox , do you have a picture of how it is broken? The easiest way is to replace the button if you are handy with soldering. If not, we can order a new PCB for you. Reach out to us vai e-mail or DM us on Discord and we can look into it, thanks!

Hey thanks for the response! I am definitely not handy enough to do such tiny soldering haha. I will probably like to get a new pcb. What's a good email for you?

**Comment by RyanH54** [comment]
Hi @sqfmi I just received my Watchy and aluminum case today and while attempting to get the aluminum case on, both buttons on the right side snapped off. The two left are okay. Is there some way I can contact you about this? Thank you.
![watchybroke](https://user-images.githubusercontent.com/85807757/121788709-54936700-cb84-11eb-85ec-05fc5a39d1d6.jpg)

**Comment by tannerburson** [comment]
I similarly got a board with a DOA button on the bottom right. No physical damage present or noticeable, the button is still tactile, but does not appear to work. Haven't had a chance to disassemble and put a multi-meter to it yet.

**Comment by sqfmi** [comment]
Hey @tannerburson @ekkofox @raywin27 @RyanH54 , you can fill out this form if you haven't already and we'll work on getting you a replacement board: https://forms.gle/hFNWrbUZxAXniciD8 . If you prefer to return your Watchy instead, please reach out to Crowd Supply and they'll help you out. We can also send you some buttons if you'd like to try replacing them. Sorry about the broken buttons!

**Comment by raywin27** [comment]
I was offered replacement buttons, and if I failed to solder them on properly a replacement board.

 

I'd like to try some SMD soldering and try and fix it myself, but if this precludes getting a new board, then I'd rather just have the already populated replacement PCB.

 

Pls let me know how you'd like me to proceed.

 

My ideal case would be to try soldering myself, and receive a replacement pcb is I stuff it up.

 

Best Regards,

 

Raymond Winchester

 
 

Sent: Monday, June 14, 2021 at 8:25 AM

**Comment by hn3000** [comment]
Hi, I already submitted the google docs form, but would also enjoy getting a set of replacement buttons to try soldering them, if that's possible -- how would I request that?

**Comment by raywin27** [comment]
last edit was 7 days ago, any update? what is a reasonable eta on shipping for the tactile switches?  I imagine you are sending us some from a cut reel.

**Comment by raywin27** [comment]
last commented by me 10 days ago, no update... so, I see three options, return the whole thing, do a chargeback, or someone updates this with an eta on when the buttons are being sent to me.

Pls let me know how to proceed.

**Comment by sqfmi** [comment]
Hi @hn3000 @raywin27 , apologies for the delay as we work through these issues. I have shipped out the replacement buttons to your addresses, please allow 3-5 days to confirm you have received them.

**Comment by RyanH54** [comment]
Hi @sqfmi for those of us that filled out the Google form for replacement boards, should we also expect those in 3-5 days?  I didn't receive any confirmation after filling out the form on June 14 so wasn't sure what to expect. Thank you!

**Comment by raywin27** [comment]
@sqfmi  i have not received any buttons

**Comment by raywin27** [comment]
are we past the charge back window?

**Comment by RyanH54** [comment]
> are we past the charge back window?

while I'm not sure about the answer to that, did you try requesting a refund through Crowd Supply? I started a ticket with them Tuesday but still waiting for a reply. I'm curious if anyone else has tried.

**Comment by raywin27** [comment]
> > are we past the charge back window?
> 
> while I'm not sure about the answer to that, did you try requesting a refund through Crowd Supply? I started a ticket with them Tuesday but still waiting for a reply. I'm curious if anyone else has tried.

I didn't I was expecting a cut reel tape of switches, but apparently that isn't coming, will call today or tomorrow, wrt the chargeback

**Comment by mikehihz** [comment]
Hi. My bottom right button also broke off shortly after booting up and running into the non-responsive right side buttons issue (which I was in the process of trying to get the Arduino IDE to see if it was a hardware or software issue. Now it's definitely a hardware issue.) Do I have any options?

**Comment by tdotfish** [comment]
Hello - I broke one of my buttons.  Not a defect, I must have just bumped it on something.  

I am very comfortable with soldering...
Is it possible to get a replacement buttons from sqfmi or is there a good site to order a replacement part?  Is LCSC a reputable vendor?  Is there a comparable part available from digikey or mouser?

**Comment by morbidick** [comment]
@rroels mentions https://www.mouser.be/ProductDetail/710-436351045816 here https://github.com/sqfmi/Watchy/issues/49#issuecomment-920834940

**Comment by aristeu** [comment]
I ordered a Watchy and when I was assembling for the first time, noticed the button was broken ("back" one, not something I could live without). Just assumed I made a mistake. After working on getting this fixed for ~2 weeks (was afraid to use smd desoldering to not damage the screen, ended up ruining the circuit line) + wait for part from China, I get it working, proper button replaced, even got a bit of epoxy to make sure the button would be mechanically sturdy. Went to assemble it, managed to break it again. I think the plastic button piece ends being pushed on top of the fragile exposed part when putting the cover on).

I'd like to suggest looking for a different button for the next hardware version if possible.

**Comment by aristeu** [comment]
TL/DR:  assembly watchy *exactly* as the picture shows, don't try to fit the board into the top part of the case first.

OK, so after getting a second watchy I managed to break a button again. Ordered a bunch of push button replacements and managed to fix both watchys. While doing it I figured what I was doing wrong: putting the board on the top part of the case then trying to fit on the bottom part. Reason that led me to do this is because those plastic tabs that go through the board holes offer some resistance that made me afraid of damaging it (ironic). Problem is that the push buttons sit too low into the top part and when you try to fit the bottom part the plastic buttons easily get on top of the push buttons plastic part and break them in the process. Repeating the assembly very carefully I managed to reproduce the same situation that'd cause the push button to break and figured what was going on. Since then, no issues.

On the soldering: I've learned that to remove the broken button using a hotair rework smd station works best but *remove the display first* because no matter how much klapton tape you put on the board, chances of damaging the screen are big (guess who waited 2 weeks to replace the slightly damaged screen). To put the new button I strongly recommend using a regular soldering iron. While attempting to use hot air the button I ended up melting just enough for the button to be stuck.
